src/react/ZodInternationalization.ts

```diff
@@ -0,0 +1,7 @@
+import { useZodInternationalization } from './useZodInternationalization'
+
+export function ZodInternationalization() {
```

Comment:
I would be more explicit with the naming and call it `ZodInternationalizationWithReactIntl`